tests/unit/test_routes/test_api_key.py

```diff
@@ -0,0 +1,90 @@
+from app.api.auth import blacklist_key
+from .helpers import get_api_key
+
+
+def test_apikey_commit_error(
+        module_client, module_db, fake_auth_from_oc, fake_commit_error):
+    client = module_client
+
+    response = client.post('api/v1/apikey', json=dict(
+        email="test@example.org",
+        password="supersecurepassword"
+    ))
+
+    assert (response.status_code == 500)
+
+
+def test_get_api_key(module_client, module_db, fake_auth_from_oc):
```

Comment:
Somewhat nitpicky, but can we move this test up to the top? I'd like to have the positive unit tests before the negative ones.